src/Tools/InstallUpdate/Program.cs

```diff
@@ -77,26 +90,79 @@ static void Main(string[] args)
 
             if (!File.Exists(sigPath))
             {
-                Console.WriteLine("A signature file could not be found to verify this update.");
+                Console.WriteLine(Resources.Program_Main_A_signature_file_could_not_be_found_to_verify_this_update_);
                 RequestManualReinstall();
                 return;
             }
 
             if (!Utils.VerifyFile(installerPath, sigPath, pubKey))
             {
-                Console.WriteLine("The update could not be verified against the signature file.");
+                Console.WriteLine(Resources.Program_Main_The_update_could_not_be_verified_against_the_signature_file_);
```

Comment:
Further, having such close alignment between your resource names and the string itself is bound to result in weird refactoring inconsistencies.